Library/Homebrew/cmd/linkapps.rb

```diff
@@ -1,12 +1,11 @@
-# Links any Applications (.app) found in installed prefixes to /Applications
 require "keg"
 require "formula"
 
 module Homebrew
   def linkapps
-    target_dir = ARGV.include?("--local") ? File.expand_path("~/Applications") : "/Applications"
+    target_dir = linkapps_target(:local => ARGV.include?("--local"))
 
-    unless File.exist? target_dir
+    unless target_dir.directory?
```

Comment:
Agreed, but the `else` you're seeing belongs to the `if` following the `unless`. This `unless` block has no `else` branch. :wink: